compaction/compaction_manager.hh

```diff
@@ -292,6 +292,7 @@ private:
 
     class strategy_control;
     std::unique_ptr<strategy_control> _strategy_control;
+    condition_variable _compaction_done;
```

Comment:
can two tables safely share the condition_variable? isn't it better to make it per table in compaciton_manager::compaction_state ?

Reply:
good catch.  They can, but t doesn't help the waiter if it's being signaled when compaction is done on another table.